### PR TITLE
fixing tl? allignment and tl* output

### PR DESCRIPTION
**Pull request by oddcoder**

### CODE REVIEW

File: libr/core/cmd_type.c [hunk 1]
```diff
@@ -61,7 +61,7 @@ static int sdbdeletelink(void *p, const char *k, const char *v) {
 }
 static int linklist(void *p, const char *k, const char *v){
 	if(!strncmp(k,"link.",strlen("link.")))
-		r_cons_printf("tl %s @ %s\n",v,k+strlen("link."));
+		r_cons_printf("tl %s @ 0x%s\n",v,k+strlen("link."));
```
Comment (radare) [inline libr/core/cmd_type.c]: missing spaces, this line needs some air

File: libr/core/cmd_type.c [hunk 2]
```diff
@@ -61,7 +61,7 @@ static int sdbdeletelink(void *p, const char *k, const char *v) {
 }
 static int linklist(void *p, const char *k, const char *v){
 	if(!strncmp(k,"link.",strlen("link.")))
```
Comment (radare) [inline libr/core/cmd_type.c]: this one as well

Comment (radare) [inline libr/core/cmd_type.c]: too many strlens of the same thing use `const int link_len = strlen ("link.");` or so

File: libr/core/cmd_type.c [hunk 3]
```diff
@@ -61,7 +61,7 @@ static int sdbdeletelink(void *p, const char *k, const char *v) {
 }
 static int linklist(void *p, const char *k, const char *v){
```
Comment (radare) [inline libr/core/cmd_type.c]: another missing space

File: libr/core/cmd_type.c (hunk 2 above)
Comment (oddcoder) [inline libr/core/cmd_type.c]: not the same function!

**Comment by oddcoder** [comment]
done adding spaces then running indent.sh afterwards

File: libr/core/cmd_type.c [hunk 4]
```diff
@@ -339,14 +341,28 @@ static int cmd_type(void *data, const char *input) {
...
+			sdb_foreach (core->anal->sdb_types, linklist, core);
 			break;
 		}
 		break;
+	case 'p': {
+		const char *type = input + 2;
+		char *ptr = strchr (type, ' ');
+		*ptr++ = 0;
+		ut64 addr = r_num_math (core->num, ptr);
+		int len = strlen ("link.") + 8;
```
Comment (XVilka) [inline libr/core/cmd_type.c]: You don't need to link before casting, cut those out

**Pull request closed without merging**